### ISSUE TITLE
Conversion to TFlite

### ISSUE THREAD

**Issue by nahla-naz** [shubham0204/OnDevice-Face-Recognition-Android#2]
Hi,

**How do you convert models to TFlite?** I tried converting, but my app is crashing while trying to get output.

Do you perform optimisation/quantisation? If you do, does it reduce the model accuracy?

Please help. Thanks.

**Comment by nahla-naz** [comment]
Hi @shubham0204 , Thank you for your prompt reply.

I'm trying to convert` FaceNet` model from `keras_facenet` python library.

I am able to convert it using Tensorflow 2.17 and get output in python. But it doesn't run on android because latest version for TFlite in android is 2.16.1. I tried converting the model using lower versions of Tensorflow in python, but the models don't work.

I am getting this error while trying to run TFlite model on android: 
_Caused by: java.lang.IllegalStateException: Internal error: Unexpected failure when preparing tensor allocations: tensorflow/lite/core/subgraph.cc **BytesRequired number of elements overflowed.**_
